README.md

```diff
@@ -168,12 +168,20 @@ You may find it useful to store the environment variable configuration in a file
 easily include it in your environment
 
 ```shell
-. ~/contrast.env
+source ~/contrast.env
```

Comment:
This came up on a previous PR, and I repeated some wisdom Rob told me about `.` being more portable than `source`, but I found the opposite to be true while testing.